about.md

```diff
@@ -6,138 +6,80 @@ title: About MDAnalysis
-   S. Benthall and S. Rostrup, editors, *Proceedings of the 15th Python in
-   Science Conference*, pages 102-109, Austin, TX, 2016. SciPy.
- * <a name="MichaudAgrawal2011"></a>N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and
-   O. Beckstein. MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics
-   Simulations. *J. Comput. Chem.* **32** (2011), 2319—2327,
-   doi:[10.1002/jcc.21787](http://dx.doi.org/10.1002/jcc.21787). 
-   PMCID:[PMC3144279](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3144279/)
-
-   (If you are reading the [HTML version of the
-   paper](http://onlinelibrary.wiley.com/doi/10.1002/jcc.21787/full),
-   have a look at the [paper
-   errata]({{site.github.wiki}}/PaperErrata). The free PubmedCentral
-   manuscript
-   [PMC3144279](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3144279/)
-   has correct code, which can be copied and pasted.)
-
-Please also see the list of [included
-algorithms]({{ site.baseurl }}/pages/citations#IncludedAlgorithms) with
-additional citations.
-
-# Historical, Technical and Artistic Remarks
-
-MDAnalysis was originally inspired by the Schulten Group's
-[MDTools](http://www.ks.uiuc.edu/Development/MDTools/) for Python, and the DCD
-reading code is derived from VMD's
-[catdcd](http://www.ks.uiuc.edu/Development/MDTools/catdcd/). MDAnalysis is GPL
-licensed, except for some 3rd party code that is included under GPL-compatible
-licenses; for instance the dcd reading code is under the [UIUC Open Source
-Licence](http://www.ks.uiuc.edu/Development/MDTools/catdcd/license.html). See
-the files AUTHORS and LICENSE in the distribution for details.
-
-Some time-critical routines are written in C or [cython](http://cython.org) and
-require a working C compiler. The minimum required version of Python is
-2.7. MDAnalysis has been successfully used on Linux and Mac OS X.
 
-The **MDAnalysis 'Atom' Logo** was designed by **Christian Beckstein** and is
-licensed under a [Creative Commons Attribution-NoDerivs 3.0 Unported
-License](http://creativecommons.org/licenses/by-nd/3.0/).
+## Community
+
+**MDAnalysis** is developed and maintained as a freely available, open-source
+project by a global community of scientists. The MDAnalysis community adheres
+to our [Code of Conduct]({{site.baseurl}}/pages/conduct/) and invites everyone
+to [participate]({{site.baseurl}}/#participating) --- be it on the mailing
+lists, through issue reports, or code contributions.
+
+All *contributors* to the MDAnalysis library and its subprojects are acknowledged
+in a file called `AUTHORS` in each source code repository and in the list of
+contributions; as examples see the [`AUTHORS` file for
+mdanalysis](https://github.com/MDAnalysis/mdanalysis/blob/develop/package/AUTHORS)
+and the [contributors for
+MDAnalysis/mdanalysis](https://github.com/MDAnalysis/mdanalysis/graphs/contributors).
+
+MDAnalysis and the algorithm implemented in the library and the subprojects are
```

Comment:
plural `algorithms`

Reply:
done (by @jbarnoud in db2ed11e6aa49ee75765c364f4b54b51429998eb)